src/main/resources/data/minebound/recipes/gilded_diamond_helmet.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "###",
+    "# #",
+    "   "
```

Comment:
Remove this third and empty empty string from all the helmet recipes, even the ones that were already done. Seems like they were missed in review.